<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {border-image: url(:/images/led.JPG)};</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="LED1_BTN">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>LED1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LED2_BTN">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LED2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LED3_BTN">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LED3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LED4_BTN">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LED4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LEDALL_BTN">
    <property name="geometry">
     <rect>
      <x>564</x>
      <y>180</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
